domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -41,8 +40,8 @@
 
 builder.Services.AddOptions<DatabaseOptions>().BindConfiguration(DatabaseOptions.Prefix).ValidateDataAnnotations()
     .ValidateOnStart();
-builder.Services.AddOptions<ConnectionInvitationCleanupServiceOptions>()
-    .BindConfiguration(ConnectionInvitationCleanupServiceOptions.Prefix).ValidateDataAnnotations().ValidateOnStart();
+builder.Services.AddOptions<TransferAgreementProposalCleanupServiceOptions>()
```

Comment:
Will it make sense to move this into the startup file for transfer?

Reply:
moved